### PR TITLE
Degree entry requirements design history

### PR DESCRIPTION
**Pull request by frankieroberto**
Explaining the changes to the degree entry requirements

⏩ [Preview](https://deploy-preview-375--bat-design-history.netlify.app/publish-teacher-training-courses/degree-entry-requirements/)

### CODE REVIEW

File: app/posts/publish-teacher-training-courses/2021-06-03-degree-entry-requirements.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+{{ appScreenshots({
+  hideContents: true,
+  items: [{
+      text: "Do you require a minimum degree classificaiton?",
+      img: { src: "degree-minimum-classification.png" }
+    }]
+}) }}
+
+For non-Primary training courses, providers are told that candidates will be given generic guidance that they degree subject should match the subject they want to teach.
```
Comment (TomCrooky) [inline app/posts/publish-teacher-training-courses/2021-06-03-degree-entry-requirements.md]: they should be their

File: app/posts/publish-teacher-training-courses/2021-06-03-degree-entry-requirements.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+
+Some providers require a minimum degree classification, such as a 2:2, on top of the statutory requirement for a degree. Not meeting this can be a reason why candidates are rejected. Research has also shown that where candidates are aware of this before applying, it can take them a long time to sift through courses in order to find those that will accept their degree classification.
+
+Currently, the requirement for a degree, any minimum classification, and any subject requirements, are all described by providers within a single text box labelled "Qualifications needed".
+
+In order to enable candidates to more easily find courses which will accept their degree classification, we are changing this text box to a set of structured questions and answers.
+
+These questions have to be answered for each course separately, as analysis research showed that whilst most providers have the same degree requirements across all their courses, some have different degree entry requirements depending on the course type.
```
Comment (TomCrooky) [inline app/posts/publish-teacher-training-courses/2021-06-03-degree-entry-requirements.md:15]: Could add : this was also supported by training providers during user research, who gave examples of degree requirements varying between courses, for example subjects without SKE courses.

File: app/posts/publish-teacher-training-courses/2021-06-03-degree-entry-requirements.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      text: "Degree requirements summarised",
+      img: { src: "degree-requirements-summarised.png" }
+    }]
+}) }}
+
+## User research findings
+
+These changes were usability tested with 5 providers in May 2021. Most participants were able to answer the new questions with ease, although some indicated they would need to check their policy with colleagues.
+
+There was broad agreement that the guidance indicating that candidates would be ‘discouraged but not blocked’ from applying if they did not meet the minimum degree classification was a good balance, as they did consider exceptional cases.
```
Comment (TomCrooky) [inline app/posts/publish-teacher-training-courses/2021-06-03-degree-entry-requirements.md]: as all providers described examples where they would be flexible in certain cases or exceptional candidates.

Comment (frankieroberto) [reply app/posts/publish-teacher-training-courses/2021-06-03-degree-entry-requirements.md]: Thanks!

**Pull request merged**